screens/Profile/Profile.jsx

```diff
@@ -7,15 +7,19 @@ import Posts from './page/Posts/Posts';
 import requestCameraPermission from '../../utils/CameraPermissionUtils';
 import {launchImageLibrary} from 'react-native-image-picker';
 import styles from './style';
-import SkeletonPosts from './page/Skeleton/SkeletonPosts';
-
+import SkeletonPosts from '../../components/Skeleton/SkeletonPosts';
+import {useEffect} from 'react';
 
 const Profile = ({navigation}) => {
   const [showHide, setShowHide] = useState(false);
   const [imageUrl, setImageUrl] = useState('');
   const {feedData} = useContext(DNAdataContext);
   const [loading, setLoading] = useState(true);
 
+  useEffect(() => {
+    setTimeout(() => setLoading(false), 2500);
```

Comment:
if it's a temporary solution until the main logic will be implemented please add TODO comment with the details. If no, please explain why you need this timeout?

Reply:
Added a `TODO` comment.